gocourse04/main.go

```diff
@@ -0,0 +1,168 @@
+package main
+
+import "fmt"
+
+// Наприклад, зоопарк ділиться на території: копитні, пернаті, примати… та інше.
+
+const (
+	Ungulates AnimalType = "Ungulates"
+	Feathered AnimalType = "Feathered"
+	Mammals   AnimalType = "Mammals"
+	Horses    Breed      = "Horses"
```

Comment:
Між групами констант бажано ставити пустий рядок. Так більш читабельніше: 
```suggestion
	Mammals   AnimalType = "Mammals"

	Horses    Breed      = "Horses"
```